main-app/app/scripts/services/game-model.js

```diff
@@ -19,7 +19,21 @@
                     function (data, status) {
                         alert('Server Error:' + status + ' information ' + data);
                     });
-            };
+
+            if (this.player2 === 'human') {return currentPlayer == player2;
```

Comment:
What happens if you select two human players? 
Who actually makes the first move.